V/ViZDoom/build_tarballs.jl

```diff
@@ -0,0 +1,67 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+# See https://github.com/JuliaLang/Pkg.jl/issues/2942
+# Once this Pkg issue is resolved, this must be removed
+uuid = Base.UUID("a83860b7-747b-57cf-bf1f-3e79990d037f")
+delete!(Pkg.Types.get_last_stdlibs(v"1.6.3"), uuid)
+
+julia_versions = [v"1.6.3", v"1.7.0", v"1.8.0", v"1.9.0"]
+
+name = "ViZDoom"
+version = v"1.1.13"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/mwydmuch/ViZDoom/archive/refs/tags/$version.tar.gz", "e379a242ada7e1028b7a635da672b0936d99da3702781b76a4400b83602d78c4"),
+    DirectorySource("./bundled"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/ViZDoom-*
+mv ../CMakeLists.txt ../ViZDoomJuliaModule.cpp src/lib_julia/
+mkdir build
+cd build
+cmake \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_FIND_ROOT_PATH=${prefix} \
+    -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_CROSSCOMPILING=FALSE \
+    -DJulia_PREFIX=${prefix} \
+    -DBUILD_JULIA=ON \
+    ..
+cmake --build . --parallel ${nproc}
+cp bin/libvizdoomjl.so bin/vizdoom bin/vizdoom.pk3 ../src/freedoom2.wad ${libdir}
+mv ../scenarios $prefix/
+install_license $WORKSPACE/srcdir/ViZDoom-*/README.md
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+include("../../L/libjulia/common.jl")
+platforms = [Platform("x86_64", "linux")] #vcat(libjulia_platforms.(julia_versions)...)
```

Comment:
This is wrong, you aren't building for the different versions of Julia anymore.

Reply:
Shoot, that was for local testing :)

Reply:
For local testing you never have to modify the list of platforms, you can pass the platform you want to build for as argument to the script: https://docs.binarybuilder.org/v0.5/#Manually-create-or-edit-build_tarballs.jl